### PR TITLE
testing figure links

### PR DESCRIPTION
**Pull request by jharenza**
### Purpose

- add versioned Figs 1, S1-S4
- update internal links to supp figures
- add a defined size (7" width) to all figures
- <s>update main internal table links</s>

### Issue

partially #239 

#### Which areas should receive a particularly close look?
NA

#### Is there anything that you want to discuss further?
Is the width of the figures fine?  

#### Is the pull request ready for review?
yes

### Pull review checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [x] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#manubot-markdown)]
- [x] All citations follow the [Manubot citation instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#citations)
- [x] All changes or additions to tables follow the [Manubot table instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#tables)
- [x] All figures follow the [Manubot figure instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#figures)
- [x] There are no new spelling errors identified by the [Manubot spellchecker](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#spellchecking)

### Spellcheck Step

The dictionary used for spellchecking can be updated.
Edit the file in `build/assets/custom-dictionary.txt` by adding new entries to the end.
You do not need to change anything else.
However, if you want to update the first line to have an accurate count of words and you want to remove non-unique ones, run the following command from within `build/assets` on your favorite OS X or Linux machine:
```
(( len = $(awk '!a[$0]++' < custom-dictionary.txt | wc -l ) - 1 )); tmpfile="$(mktemp)"; echo "personal_ws-1.1 en $len utf-8" > $tmpfile; tail -n +2 custom-dictionary.txt | awk '!a[$0]++' >> $tmpfile; mv $tmpfile custom-dictionary.txt
```

### CODE REVIEW

**Comment by jharenza** [comment]
@jaclyn-taroni this is ready for review for the figures - I was also testing a number of different ways to link the tables, but none are working, so I will just push back to how we had it before, which was simply writing "Table 1", "Table 2".

To document:

Tried, but they all only rendered the word **table** and not the number. Also tried adding a tag `{#tbl:T1 tag="1"}`, but that did not help.
`+@tbl:T1`
`@tbl:T1`
`{@tbl:T1}`

Will open a new issue for tables, unless we don't want to bother with internal links.

**Comment by jaclyn-taroni** [comment]
I'm not sure I totally followed you on the tables, but I think you might want to try an approach that's similar to when you reference different _panels_ for figures: 

```
{@tbl:T1} 1
```

File: content/03.results.md [hunk 1]
```diff
@@ -3,17 +3,17 @@
...
...Data was harmonized by the Kids First Data Resource Center using an Amazon S3 framework within CAVATICA. B, Processed data was released through the OpenPBTA GitHub project, through which contributors filed issues and developed code for analyses within docker. Additional analysis details can be found in **Figure {@fig:S1}**. Simultaneously, this manuscript was written in GitHub using Manubot. All analysis and manuscript code was reviewed and continuously integrated using CircleCI. C, Stacked bar plot summary of the number of biospecimens per phase of therapy per broad histology (GNG = ganglioglioma, LGG = low-grade glioma, DIPG = diffuse intrinsic pontine glioma, DMG = diffuse midline glioma, EPN = ependymoma, HGG = high-grade glioma, ATRT = atypical teratoid rhabdoid tumor, CNS-ET = CNS embryonal tumor, MB = medulloblastoma, PNF = plexiform neurofibroma, DNET = dysembryoplastic neuroepithelial tumor, CRANIO = craniopharyngioma, EWS = Ewing sarcoma, CPP = choroid plexus papilloma. Panel A created with BioRender.com](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/e68062fc6452287a5a2782d027055ce3e7e93d78/figures/pngs/figure1.png?sanitize=true){#fig:Fig1 width="7in"}
```
Comment (jaclyn-taroni) [inline content/03.results.md:16]: Perhaps you tested this already:

```suggestion
![**Overview of the OpenPBTA Project.** A, The Children's Brain Tumor Network and the Pacific Pediatric Neuro-Oncology Consortium collected tumor samples from 943 patients. To date, 38 cell lines were created from tumor tissue and over 2000 specimens were sequenced (N = 1035 RNA-Seq, N = 940 WGS, and N = 32 WXS or Targeted panel). Data was harmonized by the Kids First Data Resource Center using an Amazon S3 framework within CAVATICA. B, Processed data was released through the OpenPBTA GitHub project, through which contributors filed issues and developed code for analyses within docker. Additional analysis details can be found in **Figure {@fig:S1}**. Simultaneously, this manuscript was written in GitHub using Manubot. All analysis and manuscript code was reviewed and continuously integrated using CircleCI. C, Stacked bar plot summary of the number of biospecimens per phase of therapy per broad histology (GNG = ganglioglioma, LGG = low-grade glioma, DIPG = diffuse intrinsic pontine glioma, DMG = diffuse midline glioma, EPN = ependymoma, HGG = high-grade glioma, ATRT = atypical teratoid rhabdoid tumor, CNS-ET = CNS embryonal tumor, MB = medulloblastoma, PNF = plexiform neurofibroma, DNET = dysembryoplastic neuroepithelial tumor, CRANIO = craniopharyngioma, EWS = Ewing sarcoma, CPP = choroid plexus papilloma. Panel A created with [BioRender.com](https://biorender.com/)](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/e68062fc6452287a5a2782d027055ce3e7e93d78/figures/pngs/figure1.png?sanitize=true){#fig:Fig1 width="7in"}
```

File: content/03.results.md [hunk 2]
```diff
@@ -22,8 +22,8 @@ More recently, in 2016 and 2021 [@doi:10.1007/s00401-016-1545-1; @doi:10.1093/ne
...
 Importantly, this collaborative molecular subtyping process allowed us to identify data entry errors (e.g., an ETMR entered as a medulloblastoma) and mis-identified specimens (e.g., Ewing sarcoma sample labeled as a craniopharyngioma), update diagnoses using modern terms (e.g., primitive neuro-ectodermal tumor [PNET] diagnoses), and discover rarer tumor entities within the OpenPBTA (e.g., H3-mutant ependymoma, meningioma with _YAP1::FAM118B_ fusion).
-**Table 1** {#tbl:subtypes} lists the subtypes we defined within the OpenPBTA, comprising of LGATs (N = 569), high-grade gliomas (N = 314), embryonal tumors (N = 229), ependymomas (N = 65), tumors of sellar region (N = 51), mesenchymal non-meningothelial tumors (N = 21), glialneuronal tumors (N = 20), and chordomas (N = 12). 
-or detailed methods, see **STAR Methods** and **Figure S1**. 
+**Table 1** lists the subtypes we defined within the OpenPBTA, comprising of LGATs (N = 569), high-grade gliomas (N = 314), embryonal tumors (N = 229), ependymomas (N = 65), tumors of sellar region (N = 51), mesenchymal non-meningothelial tumors (N = 21), glialneuronal tumors (N = 20), and chordomas (N = 12).
```
Comment (jaclyn-taroni) [inline content/03.results.md:25]: What happens if you (leaving your tag in), try:

```suggestion
** *@tbl:T1 ** lists the subtypes we defined within the OpenPBTA, comprising of LGATs (N = 569), high-grade gliomas (N = 314), embryonal tumors (N = 229), ependymomas (N = 65), tumors of sellar region (N = 51), mesenchymal non-meningothelial tumors (N = 21), glialneuronal tumors (N = 20), and chordomas (N = 12). 
```

Looking at [these docs](https://github.com/tomduck/pandoc-tablenos#clever-references)

File: content/07.supplemental.md [hunk 3]
```diff
@@ -1,35 +1,14 @@
...
-(B) Proportion of Signature 1 plotted by phase of therapy for each cancer group.
-
-**Figure S5. Related to Figure 4**.
-(A) Receiver Operating Characteristic for _TP53_ classifier run on FPKM of poly-A RNA-Seq samples. 
-Correlation plots for telomerase scores (EXTEND) with RNA expression of _TERT_ (B) and _TERC_ (C).First two dimensions from UMAP of sample transcriptome data with points colored by `molecular_subtype` for medulloblastoma (D), ependymoma (E), low-grade glioma (F), and high-grade diffuse astrocytic tumors (G). 
+![**OpenPBTA Project Workflow, Related to Figure 1.** Biospecimens and data were collected by CBTN and PNOC. Genomic sequencing and harmonization (orange boxes) were performed by the Kids First Data Resource Center (KFDRC). Analyses in the green boxes were performed by contributors of the OpenPBTA project. Output files are denoted in blue. Figure created with [BioRender.com](BioRender.com).](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/0c4a7e10bab352d7d1b8840d1559f68bdb46bf66/figures/pngs/figureS1.png?sanitize=true){#fig:S1 tag="S1" width="7in"}
```
Comment (jaclyn-taroni) [inline content/07.supplemental.md:3]: Out of curiosity, why are there different commits being used?

File: content/03.results.md [hunk 4]
```diff
@@ -127,9 +127,9 @@ We observed 25% (15/60) of ependymoma tumors to be _C11orf95::RELA_ (now, _ZFTA:
...
... A, Low-grade astrocytic tumors (N = 227): low-grade glioma astrocytoma (N = 187), ganglioglioma (N = 35), subependymal giant cell astrocytoma (N = 2), diffuse fibrillary astrocytoma (N = 1), pilocytic astrocytoma (N = 1), and pleomorphic xanthoastrocytoma (N = 1); B, Embryonal tumors (N = 128): medulloblastomas (N = 95), atypical teratoid rhabdoid tumors (N = 24), embryonal tumors with multilayer rosettes (N = 2), other CNS embryonal tumors (N = 5), ganglioneuroblastoma (N = 1), and CNS neuroblastoma (N = 1); C, Diffuse astrocytic and oligodendroglial tumors (N = 61): diffuse midline gliomas (N = 34) and non-midline high-grade gliomas (N = 26), oligodendroglioma (N = 1); D, Other CNS tumors (N = 195): ependymomas (N = 60), dysembryoplastic neuroepithelial tumors (N = 19), meningiomas (N = 17), schwannoma (N = 11), neurofibroma plexiform (N = 7). Other CNS tumors with N < 5 are displayed in **Figure {@fig:S3}A**. Patient sex (`germline_sex_estimate`) and tumor histology (`cancer_group`) are displayed as annotations at the bottom of each plot. Only samples with mutations in the listed genes are shown. Multiple CNVs are denoted as a complex event.](images/main/Figure2.png){#fig:Fig2}
```
Comment (jaclyn-taroni) [inline content/03.results.md:132]: This shows up in the diff but I can't tell that anything has changed... 🤔

File: content/03.results.md (hunk 2 above)
Comment (jharenza) [reply content/03.results.md:25]: Didn't try with spaces, but when I tried without, it appeared to change the formatting, with or without curly braces. I can try this.

File: content/07.supplemental.md (hunk 3 above)
Comment (jharenza) [reply content/07.supplemental.md:3]: Do you mean instead of no commit? The docs said that if outside of this repo, to use a versioned figure, so I pull each latest commit as the version. That alone didn't work even though the instructions suggested it did. What worked was to add the raw content piece as you suggested, even though the docs suggest that's svg-specific.

**Comment by jharenza** [comment]
> Operating under the assumption that there were no substantive changes to the legends themselves because I did not check that thoroughly 👍🏻 
> 
> 
> 
> I am returning a few things we might try and a question.

Didn't change legends except for the figure 1 internal S1 link and that I put all supp legends on one line so the plots would render.

File: content/07.supplemental.md (hunk 3 above)
Comment (jaclyn-taroni) [reply content/07.supplemental.md:3]: The _latest commit_ in the analysis repository is `489d729cde4318654a26b0a9ef0dee6d706352cd`. The URLs for these figures don't use the same commit hash. 

This one uses: `0c4a7e10bab352d7d1b8840d1559f68bdb46bf66` 
S3 uses: `b78c63b52fe252ab26b3141272031d4c35504f6e`

Is there a reason for that?

Comment (jharenza) [reply content/07.supplemental.md]: Ok, I was navigating to the wrong area, fixed in https://github.com/AlexsLemonade/OpenPBTA-manuscript/pull/244/commits/1f716721374972b98944911c71ebfa8bddbc7d8f

File: content/03.results.md (hunk 4 above)
Comment (jharenza) [reply content/03.results.md:132]: I didn't change this, so not sure...?

File: content/03.results.md (hunk 1 above)
Comment (jharenza) [reply content/03.results.md]: update: this actually does work even though it doesn't appear it would in sublime text. pushed in https://github.com/AlexsLemonade/OpenPBTA-manuscript/pull/244/commits/149329f186fc73f2c72812a329b97d465735c612

Comment (jharenza) [reply content/03.results.md]: OK, just kidding on this - it creates a hyperlink, but the url goes to `file:///converted/BioRender.com`, so that won't work. Sigh.

File: content/03.results.md (hunk 2 above)
Comment (jharenza) [reply content/03.results.md]: Still getting `??` so will open a new issue for this.
